c3.py

```diff
@@ -243,7 +252,11 @@ def column(self, column_i):
         elif typ == SQLITE_TEXT:
             data_ptr = lib.sqlite3_column_text(self.stmt_ptr, column_i)
             data_len = lib.sqlite3_column_bytes(self.stmt_ptr, column_i)
-            return bytes(string_at(data_ptr, data_len)).decode()
+            b = bytes(string_at(data_ptr, data_len))
+            try:
+                return b.decode()
+            except UnicodeDecodeError as e:
+                raise ValueError(f"Could not decode: {b}: {e}") from e
```

Comment:
The `b` can potentially be large, so including it in the error message verbatim might not be a good idea. Did you need this change to help with debugging?

Reply:
good catch, changed this one. I just needed to understand the error, sqlite test suite provides some bogus values and expect it to fail.